addons/udes_common/models/models.py

```diff
@@ -84,3 +85,23 @@ def statistics(self, cache=False):
 def trace(self, filter=None, max=None):
     """Trace database queries"""
     return tools.QueryTracer(self.env.cr, filter=filter, max=max)
+
+
+@add_if_not_exists(models.BaseModel)
+def selection_value_displayname(self, selection_field_name):
```

Comment:
Super nit (well two): I think we can shorten this to `selection_display_name`

1. Don't think value is needed
2. Think `display_name` is tidier than `displayname`